### PR TITLE
Adding group principals to new dav endpoint

### PR DESCRIPTION
**Pull request by DeepDiver1975**
bildschirmfoto von 2016-01-08 12-12-30

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@LukasReschke @PVince81 @schiesbn @rullzer @icewind1991 please review and test - THX

**Comment by icewind1991** [comment]
Works and looks good :+1: 

But I would expect it to list users when I propfind a group

**Comment by DeepDiver1975** [comment]
> But I would expect it to list users when I propfind a group

there is a todo in the code for this - next PR ;-)

I mainly need to get this in so that we can add groups shares to calendar and contacts

File: apps/dav/tests/unit/phpunit.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 		 timeoutForLargeTests="900"
 		>
 	<testsuite name='unit'>
-		<directory suffix='test.php'>.</directory>
+		<directory suffix='.php'>.</directory>
```
Comment (PVince81) [inline apps/dav/tests/unit/phpunit.xml]: intentional ?

**Comment by PVince81** [comment]
Code looks good :+1:

File: apps/dav/tests/unit/phpunit.xml (hunk 1 above)
Comment (DeepDiver1975) [inline apps/dav/tests/unit/phpunit.xml]: yes

**Pull request merged**